stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -419,6 +420,38 @@ public Token createTokenSynchronous(final Card card, String publishableKey)
     }
 
     /**
+     *  Starts polling the {@link Source} object with the given ID. For payment methods that require
+     *  additional customer action (e.g. authorizing a payment with their bank), polling
+     *  allows you to determine if the action was successful. Polling will stop and the
+     *  provided callback will be called once the source's status is no longer
+     *  {@link Source#PENDING}, or if the given timeout is reached and the source is still
+     *  `pending`. If polling stops due to an error, the callback will be fired with the latest
+     *  retrieved source and the error.
+     *
+     *  Note that if a poll is already running for a source, subsequent calls to `pollSource`
```

Comment:
Ok. I was planning on doing that anyhow (having an in-flight stopping mechanism), but I have a few more PRs to go before this is ready to ship, and I was hoping to do so iteratively to decrease time-to-market :)

I'll go ahead and land this one as is, probably change the comment in the next one (which will involve adding a polling example to the example application). The in-flight controls will probably be after I ship v1 mobile